jobs/sql/extractions/all_encounters.sql

```diff
@@ -0,0 +1,108 @@
+SET @partition = '${partitionNum}';
```

Comment:
This is coming across as a new file, but it reality I just renamed "all_encounters_staging" and made a few changes, which I'll describe.